src/pubsub/parity/parity.js

```diff
@@ -33,6 +33,14 @@ class Parity extends PubsubBase {
     });
   }
 
+  allAccountsInfo (callback) {
```

Comment:
Alphabetical move. (Missed it on the first round through - file however is not fully alphabetical)